tests/smoke/rest.smoke.test.ts

```diff
@@ -126,7 +125,7 @@ describe('HiFi API REST Calls', () => {
             expect(settingsJSON['global-frequency-rolloff']).toBe(null);
             expect(settingsJSON['global-attenuation']).toBe(null);
             expect(settingsJSON['client-limit']).toBe(null);
-            expect(settingsJSON['max-client-limit']).toBe(20);
+            expect(settingsJSON['max-client-limit']).toBeLessThanOrEqual(200);
```

Comment:
FYI, this will be changing with the work Seth is doing in https://github.com/highfidelity/speakeasy-infra/pull/366 -- in retrospect it might be best just to pull it out if you want? Up to you and @bridie-hifi .

Reply:
oh, yeah, I meant to not include that in this PR anyway. I'll get rid of it for now butt leave a comment in case we want to add some sort of check back again after Seth's work.